sdk/monitor/azure-monitor-query/azure/monitor/query/_helpers.py

```diff
@@ -50,6 +50,10 @@ def order_results(request_order, responses):
     return ordered
 
 def construct_iso8601(start=None, end=None, duration=None):
+    try:
+        duration = 'PT{}S'.format(duration.total_seconds())
```

Comment:
Should we use isinstance?

If user uses a class that has total_seconds property, is it valid?

Reply:
What if duration is None in line 69?

Reply:
changed to `if duration is not None`

- for the second question - it's okay to be None

Reply:
Just to confirm, you mean

```py
end = Serializer.serialize_iso(end)
iso_str = None + '/' + end
```
is valid?

Reply:
oops - line 69-71 got changed when i made the changes :)
nope = `iso_str = None + '/' + end` is not a valid format - although end must be provided with a duration
- it's worth improving the message :)